[2024-10-04T00:02:53.095+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-04T00:02:53.230+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: process_employees.get_data manual__2024-10-03T23:56:21.453135+00:00 [queued]>
[2024-10-04T00:02:53.319+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: process_employees.get_data manual__2024-10-03T23:56:21.453135+00:00 [queued]>
[2024-10-04T00:02:53.320+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-04T00:02:53.958+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): get_data> on 2024-10-03 23:56:21.453135+00:00
[2024-10-04T00:02:54.078+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'process_employees', 'get_data', 'manual__2024-10-03T23:56:21.453135+00:00', '--job-id', '232', '--raw', '--subdir', 'DAGS_FOLDER/etl_prueba.py', '--cfg-path', '/tmp/tmphgnunrk5']
[2024-10-04T00:02:54.110+0000] {standard_task_runner.py:91} INFO - Job 232: Subtask get_data
[2024-10-04T00:02:54.118+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=32102) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-04T00:02:54.119+0000] {standard_task_runner.py:63} INFO - Started process 32116 to run task
[2024-10-04T00:02:54.705+0000] {task_command.py:426} INFO - Running <TaskInstance: process_employees.get_data manual__2024-10-03T23:56:21.453135+00:00 [running]> on host 5e909a8df6c4
[2024-10-04T00:02:55.302+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='process_employees' AIRFLOW_CTX_TASK_ID='get_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-03T23:56:21.453135+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-03T23:56:21.453135+00:00'
[2024-10-04T00:02:55.313+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-04T00:02:56.090+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1103 InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.bcra.gob.ar'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
[2024-10-04T00:02:56.203+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-04T00:02:56.210+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-04T00:02:56.246+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=process_employees, task_id=get_data, run_id=manual__2024-10-03T23:56:21.453135+00:00, execution_date=20241003T235621, start_date=20241004T000253, end_date=20241004T000256
[2024-10-04T00:02:56.377+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-04T00:02:56.482+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-04T00:02:56.498+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
